sda/cmd/ingest/ingest.go

```diff
@@ -388,6 +390,20 @@ func main() {
 							continue mainWorkLoop
 						}
 
+						// Set the file's hex encoded public key
+						log.Debugln("Compute and set key hash")
```

Comment:
Will these get ignored when deployed? I don't remember the loglevel we set. Just wondering if they add any value or if they will flood our logs.